app/CommandExecutor.php

```diff
@@ -83,19 +109,49 @@ public function execute(string $command, string $cwd) : int
         return proc_close($proc);
     }
 
+    public function getOutputBuffer(): string
+    {
+        $filename = $this->getOutputFileName();
+
+        if (! file_exists($filename)) {
+            return '';
+        }
+
+        $output = file_get_contents($filename);
+
+        if ($output === false) {
+            return "Error: Unexpected failure trying to read the output file $filename.";
```

Comment:
What do you guys (@dbpolito @fabriciojs) think about creating an error prefix output?

```suggestion
            return "FWD Error: Unexpected failure trying to read the output file $filename.";
```

Or create scope for any command executed, e.g:

```
docker-compose: Error on foobar.

fwd: Error on lorem ipsum.
```

Just an idea.